backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java

```diff
@@ -217,4 +219,29 @@ public Facility markFacilityAsDeleted(@Argument UUID facilityId, @Argument boole
   public Organization deleteE2EOktaOrganizations(@Argument String orgExternalId) {
     return organizationService.deleteE2EOktaOrganization(orgExternalId);
   }
+
+  /**
+   * Used for outreach purposes - emails a CSV of org admin emails for the following type: 1.
+   * "facilities" - orgs that have facilities in the state 2. "patients" - orgs outside the state
+   * that have test results for patients whose address is in the state
+   *
+   * <p>The generated CSV is sent to the outreachMailingListRecipient email
+   *
+   * @param type "facilities" or "patients"
+   * @param state State abbreviation e.g. "NJ", "MN"
+   */
+  @MutationMapping
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public boolean sendOrgAdminEmailCSV(@Argument String type, @Argument String state) {
```

Comment:
⚠️ moved this from the `OrganizationResolver.java` file to this one since I marked this as a `mutation`

Reply:
only other change, beside moving it, is returning a `boolean` rather than an `Integer`